CHANGELOG.md

```diff
@@ -39,6 +39,7 @@
 * [ENHANCEMENT] Memberlist: optimized receive path for processing ring state updates, to help reduce CPU utilization in large clusters. #4345
 * [ENHANCEMENT] Memberlist: expose configuration of memberlist packet compression via `-memberlist.compression=enabled`. #4346
 * [ENHANCEMENT] Updated Prometheus to include changes from prometheus/prometheus#9083. Now whenever `/labels` API calls include matchers, blocks store is queried for `LabelNames` with matchers instead of `Series` calls which was inefficient. #4380
+* [ENHANCEMENT] AlertManager: Retrying AlertManager UnaryPath Requests on next replica if one fail #4422
```

Comment:
These descriptions should be understandable by a typical Cortex admin; "UnaryPath" is not in the docs so probably not meaningful.
I'm not familiar with this code but it seems silences, status and receivers are the only unary requests, so could we say that?

Reply:
make sense! done!